--- qemu-2.8.0/linux-user/syscall.c	2022-12-20 13:47:30.250483042 -0600
+++ qemu-2.8.0/linux-user/syscall.c.new	2022-12-20 14:32:17.696015186 -0600
@@ -7981,10 +7981,12 @@
 #ifdef TARGET_NR_stime /* not on alpha */
     case TARGET_NR_stime:
         {
-            time_t host_time;
-            if (get_user_sal(host_time, arg1))
+            struct timespec ts;
+            ts.tv_nsec = 0;
+            if (get_user_sal(ts.tv_sec, arg1)) {
                 goto efault;
-            ret = get_errno(stime(&host_time));
+            }
+            ret = get_errno(clock_settime(CLOCK_REALTIME, &ts));
         }
         break;
 #endif
