--- git/argp/argp-fmtstream.c	2022-12-20 15:19:46.935346735 -0600
+++ git/argp/arg-fmtstream.c.new	2022-12-20 16:11:38.356552626 -0600
@@ -149,9 +149,11 @@
 	      size_t i;
 	      for (i = 0; i < pad; i++)
 		{
+#ifdef _LIBC
 		  if (_IO_fwide (fs->stream, 0) > 0)
 		    putwc_unlocked (L' ', fs->stream);
 		  else
+#endif
 		    putc_unlocked (' ', fs->stream);
 		}
 	    }
@@ -312,9 +314,11 @@
 	      *nl++ = ' ';
 	  else
 	    for (i = 0; i < fs->wmargin; ++i)
+#ifdef _LIBC
 	      if (_IO_fwide (fs->stream, 0) > 0)
 		putwc_unlocked (L' ', fs->stream);
 	      else
+#endif
 		putc_unlocked (' ', fs->stream);
 
 	  /* Copy the tail of the original buffer into the current buffer
--- git/argp/argp-help.c	2022-12-20 15:19:46.935346735 -0600
+++ git/argp/argp-help.c.new	2022-12-20 16:12:23.317970837 -0600
@@ -1873,9 +1873,11 @@
 #endif
 	    }
 
+#ifdef _LIBC
 	  if (_IO_fwide (stream, 0) > 0)
 	    putwc_unlocked (L'\n', stream);
 	  else
+#endif
 	    putc_unlocked ('\n', stream);
 
 #if _LIBC || (HAVE_FLOCKFILE && HAVE_FUNLOCKFILE)
